packages/ui/lib/components/Select/Select.tsx

```diff
@@ -102,14 +104,24 @@ export const Select = <T extends unknown>({
   customOption = false,
   disabled: fieldDisabled = false,
   loading = false,
+  showMoreButton = false,
+  showMoreAfter = 6,
```

Comment:
Do we really need this? We should only show this option as the last one..

Reply:
Actually let's use a new prop on the select, Rather called "moreOptions" and merge with the options when applicable.